### PR TITLE
#732 Swagger doc update 

### PR DESCRIPTION
**Pull request by CaitlinOCallaghan**
Various changes to wording to improve clarity and styling so that page better matches PC theme. Changes in swagger-ui.css are for color and to set display to "none" for unwanted features such as the search bar.

### CODE REVIEW

File: src/server/routes/enrichment-rest.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ var swaggerDefinition = {
   info: {
     title: 'Enrichment Services',
     version: '1.0.0',
-    description: 'This is a sample enrichment service server. You can find detailed documentation at [Wiki](https://github.com/PathwayCommons/app-ui/wiki/Enrichment-Services)',
+    description: 'This is a sample enrichment service server. More detailed documentation can be found at [Wiki](https://github.com/PathwayCommons/app-ui/wiki/Enrichment-Services).',
```
Comment (jvwong) [inline src/server/routes/enrichment-rest.js]: May we remove/change the word 'sample'.

Comment (jvwong) [reply src/server/routes/enrichment-rest.js]: Also let's drop the wiki reference. This should be where info is.

File: src/server/routes/enrichment-rest.js [hunk 2]
```diff
@@ -21,15 +21,15 @@ var swaggerDefinition = {
   "tags": [
     {
       "name": "Validation Service",
-      "description": "Validate gene list"
+      "description": "Validate gene list for proper nomenclature"
```
Comment (jvwong) [inline src/server/routes/enrichment-rest.js]: 'Validate gene list against a selection of target biological databases'

File: src/server/routes/enrichment-rest.js [hunk 3]
```diff
@@ -219,14 +219,14 @@ enrichmentRouter.post('/visualization', (req, res) => {
  *       properties:
  *         genes:
  *           type: array
- *           description: an array of genes, integers
- *             are interpreted as NCBIGENE
+ *           description: "Input genes as JSON array
```
Comment (jvwong) [inline src/server/routes/enrichment-rest.js]: 'Input gene names as an array'

File: src/server/routes/enrichment-rest.js [hunk 4]
```diff
@@ -219,14 +219,14 @@ enrichmentRouter.post('/visualization', (req, res) => {
...
+ *                        \n By default, numerical inputs will be interpreted as NCBIGENE notation unless specified, ex HGNC:11998"
```
Comment (jvwong) [inline src/server/routes/enrichment-rest.js]: 'By default, numerical inputs will be interpreted as NCBI Gene IDs.'

File: src/server/routes/enrichment-rest.js [hunk 5]
```diff
@@ -219,14 +219,14 @@ enrichmentRouter.post('/visualization', (req, res) => {
...
  *           example: ["TP53", "111", "AFF4", "111", "11998"]
  *           items:
  *             type: string
  *         target:
  *           type: string
```
Comment (jvwong) [inline src/server/routes/enrichment-rest.js]: Shouldn't this be 'targetDb'?

Comment (CaitlinOCallaghan) [reply src/server/routes/enrichment-rest.js]: The documentation changes for the new parameter names is included in the parameter pull requests

**Pull request closed without merging**